style.css

```diff
@@ -1 +1,100 @@
+body {
+  width: 1200px;
```

Comment:
If you are setting the width of the body and you want to center the page, you should put `margin: 0 auto;` on it as well.